data/json/items/tools.json

```diff
@@ -8270,5 +8270,34 @@
     "ammo": "battery",
     "initial_charges": 100,
     "max_charges": 100
+  },
+  {
+    "result": "washboard",
```

Comment:
Any reason why you moved a *recipe* into this file (which is for item definitions)? Why did you even touch that recipe at all?